docs/whatsnew/1.0.rst

```diff
@@ -9,18 +9,275 @@ What's New in SunPy 1.0?
+    >>> c = SkyCoord(-570*u.arcsec, 120*u.arcsec, obstime=start_time, frame=Helioprojective)
+    >>> new_observer = SkyCoord(lon=20*u.deg, lat=8*u.deg, radius=0.9*u.au, obstime=end_time, frame=HeliographicStonyhurst)
+    >>> solar_rotate_coordinate(c, observer=new_observer)
+    <SkyCoord (Helioprojective: obstime=2010-06-10T12:34:56.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate (obstime=2010-06-11T13:34:56.000): (lon, lat, radius) in (deg, deg, AU)
+        (20., 8., 0.9)>): (Tx, Ty, distance) in (arcsec, arcsec, km)
+        (-715.77862011, 31.87928146, 1.34122226e+08)>
+
+- **Experimental support for applying solar differential rotation to maps.**
+
+    Applying solar differential rotation to maps also accounts for changing observer position.
+    This functionality is still experimental.
+    For example, to differentially rotate a map back 23 hours::
+
+    >>> import sunpy.map
+    >>> from sunpy.data.sample import AIA_171_IMAGE
+    >>> import astropy.units as u
+    >>> from sunpy.physics.differential_rotation import differential_rotate
+    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+    >>> differential_rotate(aia, time=-23*u.hour)
+
+    `~sunpy.physics.differential_rotation.differential_rotate` also accepts a new observer keyword.
+    The amount of solar differential rotation is calculated using the time difference between the map date and observation time of the new observer.
+    For example::
+
+    >>> import sunpy.map
+    >>> from sunpy.data.sample import AIA_171_IMAGE
+    >>> import astropy.units as u
+    >>> from sunpy.physics.differential_rotation import differential_rotate
+    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+    >>> new_observer = SkyCoord(lon=-15*u.deg, lat=-4*u.deg, radius=1*u.au, obstime=aia.date-34*u.hour, frame=HeliographicStonyhurst)
+    >>> differential_rotate(aia, observer=new_observer)
+
+.. _whatsnew-1.0-maputils:
+
+Map utility functions
+=====================
+
+A new submodule `~sunpy.map.maputils` provides a number utility functions that act on `sunpy.map.GenericMap` instances.
+For example::
+
+    >>> import sunpy.map
+    >>> from sunpy.data.sample import AIA_171_IMAGE
+    >>> import astropy.units as u
+    >>> from sunpy.physics.differential_rotation import differential_rotate
+    >>> from sunpy.map import contains_full_disk, all_coordinates_from_map
+    >>> aia = sunpy.map.Map(AIA_171_IMAGE)
+    >>> contains_full_disk(aia)
+    True
+
+    >>> coordinates = all_coordinates_from_map(aia) # The coordinates for every map pixel
+    >>> coordinates.shape
+    (1024, 1024)
+
+.. _whatsnew-1.0-renamed-removed:
 
 Renamed/removed functionality
 =============================
 
-<Fill In>
+This is just some of the renamed or removed functionality.
+
+* ``sunpy.sun.sun`` functions have been re-implemented using Astropy for significantly improved accuracy. Some functions have been removed.
+* Renamed ``eccentricity_sunearth_orbit`` to ``eccentricity_sun_earth_orbit``.
```

Comment:
```suggestion
```
I subsequently deleted this function, so it's probably not worth mentioning here.